README.md

```diff
@@ -0,0 +1 @@
+Run tests only once `npm test -- --single-run`
```

Comment:
I was told that there's a README for the main repo. But I agree, this one should at least reference that one.